package/suse-migration-services-spec-template

```diff
@@ -104,9 +110,20 @@ install -D -m 644 systemd/suse-migration-console-log.service \
 # packed into a readonly squashfs layer. Therefore there is no
 # need for use of the "standard" systemd service handling macros.
 
+ls %{python3_sitelib}
```

Comment:
Do we still need this `ls` command? I think it was intended for debugging purposes?